setup.py

```diff
@@ -21,6 +21,8 @@
     # This is needed due to https://issues.apache.org/jira/browse/BEAM-3357.
     'grpcio>=1.3.5,<=1.7.3',
     'apache-beam[gcp]>=2.2',
+    'google-api-python-client>=1.6',
+    'google-cloud-bigquery>=0.25',
```

Comment:
Are you sure that version 0.25 works or should this be > 0.25 (see my TODO above)?

Reply:
Nice catch. Done. It installed v0.29.0 on my machine.